provider/data_source_keycloak_realm.go

```diff
@@ -372,6 +372,148 @@ func dataSourceKeycloakRealm() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+
+			// WebAuthn
+			"web_authn_policy": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"acceptable_aaguids": {
+							Type: schema.TypeList,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+							Optional: true,
```

Comment:
in the schema for the data source, none of these attributes should have `Optional: true` or `Default`, and all of them should have `Computed: true`

Reply:
Like this? Should `web_authn_policy` have `Optional: true`?